sonar-orchestrator/src/main/java/com/sonar/orchestrator/locator/MavenLocator.java

```diff
@@ -102,23 +117,23 @@ public File copyToFile(MavenLocation location, File toFile) {
   @Override
   @CheckForNull
   public InputStream openInputStream(MavenLocation location) {
-    File target = locateInLocalRepositoryAndLog(location);
+    File target = locate(location);
     if (target == null) {
       return null;
     }
 
-    return openFromLocalRepository(target);
-  }
-
-  private static InputStream openFromLocalRepository(File target) {
     try {
       return FileUtils.openInputStream(target);
     } catch (IOException e) {
       throw new IllegalStateException("Fail to open " + target, e);
     }
   }
 
-  private static String path(MavenLocation location) {
+  private static String pathInMavenLocalRepository(MavenLocation location) {
     return StringUtils.replace(location.getGroupId(), ".", "/") + "/" + location.getArtifactId() + "/" + location.version() + "/" + location.getFilename();
   }
+
+  private static String cacheKeyOf(MavenLocation location) {
+    return DigestUtils.md5Hex(location.toString());
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Use a stronger hashing algorithm than MD5. ([squid:S2070](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2070))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.orchestrator%3Aorchestrator-parent&branch=feature%2Fsb%2FORCH-409%2Fartifactory&issues=AWG-BSNWJBf1BWcDXJwz&open=AWG-BSNWJBf1BWcDXJwz)

[ISSUE_KEY]: AWG-BSNWJBf1BWcDXJwz